backend/controller/scaling/localscaling/local_scaling.go

```diff
@@ -51,7 +51,7 @@ type localScaling struct {
 
 type devModeRunner struct {
 	uri url.URL
-	// Set to None under mysterious circumstances...
+	// The deployment key of the deployment that is currently running
```

Comment:
But why can it be None?

Reply:
Because the runner has stopped for whatever reason. This would normally be a transient condition, unless the runner is in a crash loop and aborts.